src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -423,6 +424,58 @@ def assertUrl(self, url):
                     urllib2.urlopen(urllib2.Request(url))
                 except:
                     self.fail()
+                    
+        def testCleanCache(self):
+            # Make a bunch of jobs
+            master = self.master
+
+            # Create parent job
+            rootJob = master.create('rootjob', 12, 34, 35)
+            
+            # Create a bunch of child jobs
+            for i in range(100):
+                child = master.create("child%s" % i, 23, 45, 46, 1)
+                rootJob.stack.append(((child.jobStoreID, 23, 45, 46, 1),))
+            master.update(rootJob)
+            
+            # See how long it takes to clean with no cache
+            noCacheStart = time.time()
+            master.clean(rootJob)
+            noCacheEnd = time.time()
+            
+            noCacheTime = noCacheEnd - noCacheStart
+            
+            # See how long it takes to clean with cache
+            jobCache = {jobWrapper.jobStoreID: jobWrapper
+                        for jobWrapper in master.jobs()}
+            cacheStart = time.time()
+            master.clean(rootJob, jobCache=jobCache)
+            cacheEnd = time.time()
+            
+            cacheTime = cacheEnd - cacheStart
+            
+            print("Without cache: %s With cache: %s" % (str(noCacheTime), str(cacheTime)))
```

Comment:
The syntax is valid but the parentheses are redundant and will be eliminated. It just looks as if you are using the print function. I would just remove the parentheses. I would only use the `print()` function if wanted access to its `file=` keyword argument. Choosing the output stream is the only feature that the `print` statement doesn't give you. I think we mix and match print statement and function in Toil, and I don't see a problem with that as long as we don't use visual ambiguity introduced by redundant parentheses.